SlimApp/Trainer.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Trainer.swift
+//  SlimApp
+//
+//  Created by Aleksandr Shapovalov on 19/01/23.
+//
+
+import Foundation
+
+struct Trainer: Codable, Identifiable {
+    var id: UUID?
+    var name: String
+    var email: String
+    var password: String
```

Comment:
Its generally not a good idea to store the password I think. Maybe we should consider removing this property

Reply:
fixed